kernel/default.nix

```diff
@@ -99,9 +112,14 @@ buildLinux (args // {
     # Override the default CMA_SIZE_MBYTES=32M setting in common-config.nix with the default from tegra_defconfig
     # Otherwise, nvidia's driver craps out
     CMA_SIZE_MBYTES = lib.mkForce (freeform "64");
+    MODVERSIONS = no;
```

Comment:
What's this for?

Reply:
it was a left over while trying to track down the mismatch of module versions when building against mainline.